openacademy/models/models.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class Course(models.Model):
+    _name = 'openacademy.course'
+    description = fields.Text()
+
+    name = fields.Char(string="Title", required=True)
+    _description = 'Model to store course'
```

Comment:
`_name` and `_description` are attributes of the model, should be [together](https://github.com/odoo/odoo/blob/15.0/addons/account/models/account_account_tag.py#L7-L8). Please organize them.
```suggestion
    _name = 'openacademy.course'
    _description = 'Model to store course'
    
     name = fields.Char(string="Title", required=True)
    description = fields.Text()
```